samples/web/content/testrtc/js/call.js

```diff
@@ -48,15 +48,19 @@ Call.prototype = {
     this.constrainOfferToRemoveVideoFec_ = true;
   },
 
-  // When the peerConnection is closed the callback is called once returning
-  // with an array of gathered stats.
-  gatherStats: function(peerConnection, callback, interval) {
+  // When the peerConnection is closed the statsFinishedCallback is called once
+  // returning with an array of gathered stats. timeForFirstFrameCallback will
+  // be called once when googFrameRateInput is > 0 with a timestamp in
```

Comment:
Please update the documentation and fix "firstFrameCb will be called at most once when..."